superset_patchup/views.py

```diff
@@ -0,0 +1,76 @@
+"""
+Additional extensions to SuperSet views
+"""
+
+from flask_appbuilder import expose
+from flask_appbuilder.models.sqla.interface import SQLAInterface
+import superset
+
+from .version import (VERSION, __version__)
+
+
+class SupersetKetchupApiView(superset.views.base.BaseSupersetView):
+
+    """
+    Extensions to Superset API
+    """
+
+    def __init__(self):
+        self.route_base = "/superset-ketchup/api"
+        super().__init__()
+
+    @superset.views.base.api
+    @expose('/version/', methods=['GET'])
+    def version(self):
+        """Returns *superset-api* version"""
+        return self.json_response(
+            {'version': VERSION, 'versionStr': __version__})
+
+    @superset.views.base.api
+    @expose('/all_dashboards/', methods=['GET'])
+    def all_dashboards(self):
```

Comment:
can we add a decorator here to protect this endpoint?

Reply:
api_has_access?  Sure.